Lightyear/Char.idr

```diff
@@ -18,7 +18,7 @@ import Lightyear.Core
 import Lightyear.Combinators
 import Lightyear.Errmsg
 
-%access public
+%access public export
```

Comment:
probably not, I have since revised a few of my own projects and will revisit this patch to lightyear.